apps/catalog-admin/pages/styles.css

```diff
@@ -0,0 +1,381 @@
+
+    html {
+      -webkit-text-size-adjust: 100%;
+      font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+      line-height: 1.5;
+      tab-size: 4;
+      scroll-behavior: smooth;
+    }
+    body {
+      font-family: inherit;
+      line-height: inherit;
+      margin: 0;
+    }
+    h1,
+    h2,
+    p,
+    pre {
+      margin: 0;
+    }
+    *,
+    ::before,
+    ::after {
+      box-sizing: border-box;
+      border-width: 0;
+      border-style: solid;
+      border-color: currentColor;
+    }
+    h1,
+    h2 {
+      font-size: inherit;
+      font-weight: inherit;
+    }
+    a {
+      color: inherit;
+      text-decoration: inherit;
+    }
+    pre {
+      font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, Liberation Mono, Courier New, monospace;
+    }
+    svg {
+      display: block;
+      vertical-align: middle;
```

Comment:
warning fra linter: "Property is ignored due to the display. With 'display: block', vertical-align should not be used."

Reply:
Samme gjelder styles.css - kom med da man opprettet ny app, tror hele filen kan fjernes.